[2020-02-23 16:36:31,986] [INFO] [MBP][conftest.py - line:19] 自动化测试开始!
[2020-02-23 16:36:31,988] [INFO] [MBP][driver.py - line:23] 79URX19227000484
[2020-02-23 16:37:43,398] [INFO] [MBP][driver.py - line:36] 初始化driver异常!USB device 79URX19227000484 is offline
[2020-02-23 16:37:43,400] [INFO] [MBP][conftest.py - line:21] driver初始化
